globals.go

```diff
@@ -363,6 +363,10 @@ const (
 	// a heartbeat
 	HeartbeatTimeoutThreshold = 7 * time.Minute
 
+	// MaxTeardownGroupThreshold specifies the duration after which the host should no longer continue
+	// to tear down a task group. This is set one minute longer than the agent's maxTeardownGroupTimeout.
+	MaxTeardownGroupThreshold = 4 * time.Minute
```

Comment:
This is longer because this is for the case where something happens and the host gets stuck. In an ideal environment the timeout will be what tears it down. This should only kick in if something went really wrong, so setting it to 4 minutes gives it a buffer.